### ISSUE TITLE
kvm support not available

### ISSUE THREAD

**Issue by marianoka** [GNS3/gns3-gui#3636]
hi
I'm having an issue. the thing is kvm support is not supported under mac fusion running on mac m2 sequioa . What can I do to enable this feature. I can't add any appliance.
Pls help
Thanks
Mariano

**Comment by smart2128** [comment]
This is not a bug.
KVM is a Linux kernel virtualization feature, not native to MacOS.
For best performances, you should probably use the GNS3-VM.
You can find further information on the GNS3 official documentation.

**Comment by grossmj** [comment]
This is correct, please use the GNS3 VM (there are limitations on Apple Silicon though) or a remote Linux computer to run a GNS3 server.
